base/system/winlogon/sas.c

```diff
@@ -802,7 +822,13 @@ HandleLogoff(
         ERR("Failed to allocate mem for thread data\n");
         return STATUS_NO_MEMORY;
     }
-    LSData->Flags = Flags;
+
+    LSData->Flags = EWX_LOGOFF;
+    if (wlxAction == WLX_SAS_ACTION_FORCE_LOGOFF)
+    {
+        LSData->Flags |= EWX_FORCE;
+    }
```

Comment:
```suggestion
        LSData->Flags |= EWX_FORCE;
```

Reply:
The suggestion kept the LSData->Flags stuff grouped.

Reply:
Yes, it's still grouped, but I would prefer the current style because if we add more flags in the future, the code could be located easier.